brion/plugin/spikeReportBinary.cpp

```diff
@@ -199,24 +169,22 @@ Spikes SpikeReportBinary::readUntil( const float max )
         pushBack( spikeArray[_startIndex], spikes );
     }
 
-    if ( _startIndex == nElems )
+    if( _startIndex == nElems )
     {
-        _currentTime = UNDEFINED_TIMESTAMP;
+        _currentTime = spikeArray[nElems - 1].first +
```

Comment:
Same comment as above : access to an empty array when _startIndex == nElems == 0